src/odemis/util/test/dataio_test.py

```diff
@@ -223,6 +223,60 @@ def test_splitext(self):
             self.assertEqual(ao, eo, "Unexpected output for '%s': %s" % (inp, ao))
 
 
+class TestSplitPlanes(unittest.TestCase):
+    def _input_image_2d(self):
+        metadata = {model.MD_SW_VERSION: "1.0-test",
+                    model.MD_HW_NAME: "fake hw",
+                    model.MD_DESCRIPTION: "sem",
+                    model.MD_DIMS: "XY"}
+
+        return model.DataArray(numpy.zeros((512, 512), dtype=numpy.dtype("uint16")), metadata.copy())
+
+    def _input_image_3d(self):
+        metadata = {model.MD_SW_VERSION: "1.0-test",
+                    model.MD_HW_NAME: "fake hw",
+                    model.MD_DESCRIPTION: "sem",
+                    model.MD_DIMS: "CTZYX",
+                    }
+
+        return model.DataArray(numpy.zeros((3, 8, 10, 256, 512), dtype=numpy.dtype("uint16")), metadata.copy())
+
+    def test_2d_plane(self):
+        """Test that for a 2d DataArray the data is returned unchanged."""
+        two_dim_da = self._input_image_2d()
+        da = _split_planes(two_dim_da)[0]
+        self.assertIsInstance(da, model.DataArray)
+        self.assertEqual(da.shape, (512, 512))
+
+    def test_multi_dim_pLane_yx(self):
+        """
+        Test that for an array with more than 2 dimensions and the x- and y-axes ordered as YX,
+        the input remains unchanged, and the output is correct.
+        """
+        multi_dim_da = self._input_image_3d()
+        list_da = _split_planes(multi_dim_da)
+        self.assertIsInstance(list_da, list)
+        for da in list_da:
+            self.assertIsInstance(da, model.DataArray)
+            self.assertEqual(da.shape, (256, 512))
+            self.assertEqual(da.metadata[model.MD_DIMS], "YX")
+            self.assertEqual(multi_dim_da.metadata[model.MD_DIMS], "CTZYX")
+
+    def test_multi_dim_pLane_xy(self):
+        """
+        Test that for an array with more than 2 dimensions and the x- and y-axes ordered as XY,
+        the input remains unchanged, and the output is correct.
+        """
+        multi_dim_da = self._input_image_3d()
+        multi_dim_da.metadata[model.MD_DIMS] = "CTZXY"
```

Comment:
Why is the MD_DIMS metatdata set to "CTZXY" here when this is already done in self._input_image_3d?